### PR TITLE
Resolving code/logic issues in material UI tests

### PR DESCRIPTION
**Pull request by Innokentiy945**

### CODE REVIEW

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java [hunk 1]
```diff
@@ -17,14 +17,4 @@ public void defaultButtonTest() {
...
-
-    @Test(expectedExceptions = {RuntimeException.class},
-            expectedExceptionsMessageRegExp = ".*(Can't perform click. Element is disabled)")
-    public void disabledButtonTest() {
-        inputButtonDisabledPage.shouldBeOpened();
-
-        buttonFrame.button.is().disabled();
-        buttonFrame.button.click();
-    }
-
```
Comment (evasileva-epam) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java:29]: Without this test no check for disabled button at all

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/FabTests.java [hunk 2]
```diff
@@ -17,14 +17,4 @@ public void defaultFabTest() {
...
-
-    @Test(expectedExceptions = {RuntimeException.class},
-            expectedExceptionsMessageRegExp = ".*(Can't perform click. Element is disabled)")
-    public void disabledFabTest() {
-        inputFabDisabledPage.open();
-
-        fabFrame.fab.is().disabled();
-        fabFrame.fab.click();
-    }
-
```
Comment (evasileva-epam) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/FabTests.java:29]: Without this test no check for disabled fab at all

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/navigation/StepperTests.java [hunk 3]
```diff
@@ -13,40 +13,40 @@
...
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("You are on Step #2");
+        stepperFrame.stepperNextButton.clickNextButton();
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("You are on Step #3");
+        stepperFrame.stepperFinishButton.clickFinishButton();
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("All steps completed");
+        stepperFrame.stepperResetButton.clickResetButton();
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("You are on Step #1");
+        stepperFrame.stepperNextButton.clickNextButton();
+        stepperFrame.stepperBackButton.clickBackButton();
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("You are on Step #1");
```
Comment (evasileva-epam) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/navigation/StepperTests.java:27]: Update with one variant for click method: click(). The same for other stepper tests

Comment (evasileva-epam) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/navigation/StepperTests.java:27]: Update name of the method' stepTitleDisplayed' to' text()' or something like this, as word displayed has association with only display on page without checking for text

**Review by evasileva-epam — CHANGES_REQUESTED** [review]

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/navigation/StepperFrame.java [hunk 4]
```diff
@@ -8,4 +8,25 @@
...
+    public Stepper stepperFinishButton;
+
+    @JDIStepper(stepperResetButton = "//*[text()='Reset']/..")
+    public Stepper stepperResetButton;
+
+    @JDIStepper(stepperSkipButton = "//*[text()='Skip']/..")
+    public Stepper stepperSkipButton;
+
+    @JDIStepper(stepperCompleteStepButton = "//*[text()='Complete Step']/..")
+    public Stepper stepperCompleteButton;
```
Comment (evasileva-epam) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/pages/navigation/StepperFrame.java:31]: Are really annotations suitable for stepper? Isn't enough simple locators?

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java (hunk 1 above)
Comment (Innokentiy945) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java:29]: Runtime errors processed in core

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/FabTests.java (hunk 2 above)
Comment (Innokentiy945) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/FabTests.java:29]: Runtime errors processed in core

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/navigation/StepperTests.java (hunk 3 above)
Comment (Innokentiy945) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/navigation/StepperTests.java:27]: done

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/navigation/StepperFrame.java (hunk 4 above)
Comment (evasileva-epam) [reply jdi-light-material-ui-tests/src/main/java/io/github/com/pages/navigation/StepperFrame.java:31]: In future it's planning to have if it's possible all controls on one page for each element, so annotations could be suitable, no sense to update now then

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java (hunk 1 above)
Comment (evasileva-epam) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java:29]: Can't see check for disabled button in ButtonTests.java

Comment (Innokentiy945) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java:29]: fixed without runtime exception

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/navigation/StepperTests.java (hunk 3 above)
Comment (Innokentiy945) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/navigation/StepperTests.java]: fixed

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/layouts/ContainerFrame.java [hunk 5]
```diff
@@ -1,4 +1,4 @@
-package io.github.com.pages.layout;
+package io.github.com.pages.layouts;
```
Comment (evasileva-epam) [inline jdi-light-material-ui-tests/src/main/java/io/github/com/pages/layouts/ContainerFrame.java:1]: Please use "layout" variant for the folders, as its in material ui site and our storybook and site

**Review by evasileva-epam — CHANGES_REQUESTED** [review]

File: jdi-light-material-ui-tests/src/main/java/io/github/com/pages/layouts/ContainerFrame.java (hunk 5 above)
Comment (Innokentiy945) [reply jdi-light-material-ui-tests/src/main/java/io/github/com/pages/layouts/ContainerFrame.java]: done

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java [hunk 6]
```diff
@@ -19,10 +19,9 @@ public void defaultButtonTest() {
     }
 
     @Test
-    public void disabledButtonTest() {
+    public void disabledButton() {
```
[the remark this answers is not in the document]
Comment (Innokentiy945) [reply jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java:22]: fixed

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/navigation/Stepper.java [hunk 7]
```diff
@@ -70,11 +55,6 @@ public void setup(Field field) {
 
         root = j.root();
         stepperTitle = j.stepperTitle();
-        stepperNextButton = j.stepperNextButton();
-        stepperBackButton = j.stepperBackButton();
-        stepperFinishButton = j.stepperFinishButton();
-        stepperResetButton = j.stepperResetButton();
-        stepperSkipButton = j.stepperSkipButton();
-        stepperCompleteStepButton = j.stepperCompleteStepButton();
+        stepperButton = j.stepperNextButton();
```
[the remark this answers is not in the document]
Comment (Innokentiy945) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/navigation/Stepper.java:58]: done

File: jdi-light-material-ui-tests/src/main/java/io/github/com/StaticSite.java [hunk 8]
```diff
@@ -41,8 +41,10 @@
 import io.github.com.pages.inputs.SplitButtonGroupFrame;
 import io.github.com.pages.inputs.SwitchFrame;
 import io.github.com.pages.inputs.TextFieldFrame;
-import io.github.com.pages.navigation.BreadcrumbsFrame;
 import io.github.com.pages.layout.ContainerFrame;
+import io.github.com.pages.navigation.BreadcrumbsFrame;
+import io.github.com.pages.layout.BoxFrame;
+import io.github.com.pages.layout.HiddenFrame;
```
[the remark this answers is not in the document]
Comment (Innokentiy945) [reply jdi-light-material-ui-tests/src/main/java/io/github/com/StaticSite.java:47]: done

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/datadisplay/TypographyAssert.java [hunk 9]
```diff
@@ -2,7 +2,7 @@
 
 import com.epam.jdi.light.asserts.generic.UIAssert;
 import com.epam.jdi.light.common.JDIAction;
-import com.epam.jdi.light.material.elements.datadispay.Typography;
+import com.epam.jdi.light.material.elements.datadisplay.Typography;
```
[the remark this answers is not in the document]
Comment (Innokentiy945) [reply jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/datadisplay/TypographyAssert.java]: done

**Pull request merged**